plugin/src/main/java/com/denizenscript/denizen/scripts/commands/world/TickCommand.java

```diff
@@ -0,0 +1,95 @@
+package com.denizenscript.denizen.scripts.commands.world;
+
+import com.denizenscript.denizencore.exceptions.InvalidArgumentsRuntimeException;
+import com.denizenscript.denizencore.objects.core.ElementTag;
+import com.denizenscript.denizencore.scripts.ScriptEntry;
+import com.denizenscript.denizencore.scripts.commands.AbstractCommand;
+import com.denizenscript.denizencore.scripts.commands.generator.*;
+import org.bukkit.Bukkit;
+import org.bukkit.ServerTickManager;
+
+public class TickCommand extends AbstractCommand {
+
+    // <--[command]
+    // @Name tick
+    // @Syntax tick [rate/step/sprint/freeze/reset] (amount:<amount>) (cancel)
+    // @Required 1
+    // @Maximum 2
+    // @Short Controls the server's tick rate.
+    // @Group world
+    //
+    // @Description
+    // Controls the server's tick rate. Versions 1.20+ only.
+    //
+    // To change the tick rate, use the 'rate' argument and input the amount using 'amount'. The tick rate must be a number between 1.0 and 10000.0 (inclusive).
+    // To reset the tick rate to the normal value (20.0), use the 'reset' argument.
+    //
+    // To freeze the tick rate, use the 'freeze' argument. To unfreeze, add the 'cancel' argument.

```

Comment:
Docs should specify better what freezing does (and, well, what any tick rate modifications do).
The wiki says `Freezes all gameplay elements, except for players and any entity a player is riding.` (but of course do not just copypaste the wiki).

Docs should specify clearly what vanilla things freeze or have rate changed (blocks? mobs? non-living entities like items? players? etc.) and what plugin-effects this has (if you freeze, does Denizen break? If you set rate 1, does the Denizen `on tick` fire only once a second? Do other common things change?)